feature-group-definitions/user-activation.yml

```diff
@@ -0,0 +1,8 @@
+name: User activation
+description: The `navigator.userActivation` API reveals whether the user has interacted with the page through an "activation" gesture such as a click, tap, or keyboard shortcut. This can be useful when working with APIs that require user activation, such as the fullscreen API.
```

Comment:
@mustaqahmed can you help me review this description? I've tried to explain the use case but am not sure I got it right. Is the main purpose of the API to predict when a call will fail, to provide feature detection for the new user activation model, or something else?

Reply:
Thanks for bringing this to my attention.  I made some suggestions above.